tools/debian_buildenv.sh

```diff
@@ -104,6 +105,7 @@ case "$1" in
             qt6-declarative-private-dev \
             qt6-base-private-dev \
             qt6-qpa-plugins \
+            qt6-wayland \
```

Comment:
This pulls Wayland in. We don't want this. Mixxx does not depend on wayland.

Reply:
This is required if you are running with `QT_QPA_PLATFORM=wayland`. Since our minimal system requirement is Wayland by default (Ubuntu 24.04), I think it would make sense to have it, but happy to move that in the Dockerfile otherwise

Reply:
Moved the Dockerfile